UnityProject/Assets/Scripts/Health/Living/PlayerHealth.cs

```diff
@@ -1,10 +1,7 @@
-﻿using System;
+﻿using Sprites;
 using System.Collections;
 using System.Collections.Generic;
-using PlayGroup;
-using Sprites;
 using UnityEngine;
-using UnityEngine.Networking;

```

Comment:
You sure network should be out?

Reply:
apparently it wasnt used, otherwise Travis would have complained, right?

Reply:
Yeah, but I just don't like sneaking them out in an unrelated PR.

Reply:
But you are right, if it gave compile error's Travis would have catched that.

I just don't like it being put into this PR... This PR doesn't have the quality to include a code cleanup, because it needs cleaning up itself...

Reply:
yes networking should be out because it is not being used therefore it does not need to be there.